src/openforms/appointments/tests/test_views.py

```diff
@@ -50,9 +50,10 @@ def test_good_token_and_submission_redirect_and_add_submission_to_session(self):
             response = self.client.get(endpoint)
 
         expected_redirect_url = (
-            furl("http://maykinmedia.nl/myform/afspraak-annuleren")
+            furl("http://maykinmedia.nl/myform")
             .add(
                 {
+                    "_action": "afspraak-annuleren",
```

Comment:
so right now with hash based routing, this would result in a URL like `example.com/myform?time=...&submission_uuid=...#/afspraak-annuleren` while it should be:

`example.com/myform#/afspraak-annuleren?time=...&submission_uuid=...` - i.e. the time & submission_uuid really should be `_action_params` as they're directly related to the action.